scrollingelement.js

```diff
@@ -36,12 +36,15 @@ if (!('scrollingElement' in document)) (function() {
 		return null;
 	}
 
+	function isStandardsMode() {
+		return /^CSS1/.test(document.compatMode);
+	}
+
 	// Note: standards mode / quirks mode can be toggled at runtime via
 	// `document.write`.
 	var isCompliantCached;
 	var isCompliant = function() {
-		var isStandardsMode = /^CSS1/.test(document.compatMode);
-		if (!isStandardsMode) {
+		if (!isStandardsMode()) {
```

Comment:
This patch executes `isStandardsMode()` twice in the worst case. Please revert to doing the standards mode check just once and storing the result in `isStandardsMode`.